rllib/core/rl_module/rl_module.py

```diff
@@ -54,7 +54,10 @@ class SingleAgentRLModuleSpec:
 
     Args:
         module_class: The RLModule class to use.
-        observation_space: The observation space of the RLModule.
+        input_space: The input space space of the RLModule. This may differ from the
+            observation space of the environment. For example, a discrete observation
+            space of an environment, would usually correspond to a one-hot encoded
+            input space for the RLModule.
```

Comment:
I've repeatedly encountered users confused about the difference between policy observation space and environment observation space. I'd like to clear this up here with this remark.